src/DeviceInterface/InterfacePrototype.js

```diff
@@ -32,10 +33,9 @@ import {initFormSubmissionsApi} from './formSubmissionsApi.js'
 class InterfacePrototype {
     attempts = 0
     /** @type {import("../Form/Form").Form | null} */
-    currentAttached = null
+    activeForm = null
```

Comment:
The previous name was rather confusing. I couldn't understand why we were not storing the form anyway. In fact we were, but I couldn't find it because of the name 🙃. This should help 🙂.